system-configuration-sys/src/dynamic_store_copy_specific.rs

```diff
@@ -0,0 +1,36 @@
+// automatically generated by rust-bindgen
+
+// Generated using:
+// bindgen 0.51.1
+// macOS SDK 10.15.
+
+use core_foundation_sys::dictionary::CFDictionaryRef;
+use core_foundation_sys::string::{CFStringEncoding, CFStringRef};
+use dynamic_store::SCDynamicStoreRef;
+
+use libc::c_uint;
+
+pub type __uint32_t = c_uint;
+pub type __darwin_gid_t = __uint32_t;
+pub type __darwin_uid_t = __uint32_t;
+pub type gid_t = __darwin_gid_t;
+pub type uid_t = __darwin_uid_t;
+pub type UInt32 = c_uint;
```

Comment:
There are still a number of types here that I commented on earlier. These are not `SCDynamicStoreCopyProxies` types. We need to blacklist them or find other ways to make them not be generated.

Reply:
While these types aren't part of `SCDynamicStoreCopyProxies`, they are just type definitions for `libc::c_unit`, so there's no harm in having those. I could of course blacklist them and add the exact same definitions using `--raw-line`, but is that really better?